File: SDKs/twilio.com/twilio_microvisor_v1/1.34.0/go/pkg/models/shared/microvisorv1device.go
package shared

import (
	"time"
)

type MicrovisorV1Device struct {
	AccountSid  *string      `json:"account_sid"`
	App         *interface{} `json:"app"`
	DateCreated *time.Time   `json:"date_created"`
	DateUpdated *time.Time   `json:"date_updated"`
	Logging     *interface{} `json:"logging"`
	Sid         *string      `json:"sid"`
	UniqueName  *string      `json:"unique_name"`
	URL         *string      `json:"url"`
}

File: SDKs/twilio.com/twilio_microvisor_v1/1.37.3/typescript/src/sdk/models/shared/microvisorv1device.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class MicrovisorV1Device extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=account_sid" })
  accountSid?: string;

  @SpeakeasyMetadata({ data: "json, name=app" })
  app?: any;

  @SpeakeasyMetadata({ data: "json, name=date_created" })
  dateCreated?: Date;

  @SpeakeasyMetadata({ data: "json, name=date_updated" })
  dateUpdated?: Date;

  @SpeakeasyMetadata({ data: "json, name=logging" })
  logging?: any;

  @SpeakeasyMetadata({ data: "json, name=sid" })
  sid?: string;

  @SpeakeasyMetadata({ data: "json, name=unique_name" })
  uniqueName?: string;

  @SpeakeasyMetadata({ data: "json, name=url" })
  url?: string;
}
